doc_src/pages/docs/plugins.md

```diff
@@ -49,9 +49,9 @@ If you don't need any plugins, or want to load plugins individually, use <code>t
 Add plugins to your project by including their js files and calling `TomSelect.define`.
 
 ```js
-import TomSelect from 'tom-select/dist/js/tom-select.base.js';
-import TomSelect_remove_button from 'tom-select/dist/js/plugins/remove_button.js';
-import TomSelect_dropdown_header from 'tom-select/dist/js/plugins/dropdown_header.js';
+import TomSelect from 'tom-select/base';
+import TomSelect_remove_button from 'tom-select/plugins/remove_button.js';
+import TomSelect_dropdown_header from 'tom-select/dropdown_header.js';
```

Comment:
IMO this is cleaner and transparently handles ESM vs. CJS. This is enabled by the new `exports` in `package.json`.